src/image.ts

```diff
@@ -47,11 +47,12 @@ export function useSanityImage(
       .width(Math.round(width))
       .quality(quality)
       .fit(fit)
+      .auto('format')
```

Comment:
Looks good, but I think we need to allow an option for this to be turned off, even if it's on by default.

- [ ] add option for auto format